### ISSUE TITLE
Database test suite is not thread-safe (?)

### ISSUE THREAD

**Issue by 23Skidoo** [genomizer/genomizer-server#136]
The database test suite works on my machine, but [fails on Travis](https://travis-ci.org/genomizer/genomizer-server/builds/58889322#L733) with the following error:

```
ERROR: duplicate key value violates unique constraint "pkey_experiment"

[junit] Detail: Key (expid)=(Exp1) already exists.
```

I suspect that the reason is a race condition in the setup code.

**Comment by yakrar** [comment]
Could also be that the test class running immediately before errors start popping up (`AddAnnotationPossibilitiesTests`) doesn't seem to be using `TestInitializer` correctly.

Most testclasses in the database testsuite call `TestInitializer.setup()` (or something equivalent) to generate a `DatabaseAccessor`, but `AddAnnotationPossibilitiesTests` seems to create it's accessor "manually" instead. I theorize this might disrupt the usual clean-up routines. 

I'll try and look in to this later today.

**Comment by 23Skidoo** [comment]
Fixed in #142.
